### PR TITLE
Add an optional id to `dunstctl close`

### PR DESCRIPTION
**Pull request by bynect**

### CODE REVIEW

**Review by zappolowski — COMMENTED** [review]
Looks good. It would be nice to adjust the completions as well (at least the summary strings) for zsh and fish as they're misleading otherwise.

**Comment by bynect** [comment]
> Looks good. It would be nice to adjust the completions as well (at least the summary strings) for zsh and fish as they're misleading otherwise.

right, forgot about that 👍🏻

**Comment by bynect** [comment]
@zappolowski I fixed some typos and updated the completions. Unfortunately we can't autocomplete the id (see my suggestion in #1348).

File: completions/_dunstctl.zshcomp
Comment (zappolowski) [inline completions/_dunstctl.zshcomp:1]: This comment applies to both completions: the descriptions for each subcommand were taken verbatim from the output of `dunstctl help` and I think it should stay this way.

Thus I suggest to adjust the output of help:
* `is-paused`: `>` → `greater than`
* `history-pop`: remove trailing period
* `help`: just `Show help` should be fine

Comment (bynect) [reply completions/_dunstctl.zshcomp:1]: Sure. I noticed that the help message itself had some typos.

**Comment by bynect** [comment]
done.

unrelated question, should we add a completion for dunstify later?

**Review by zappolowski — APPROVED** [review]

**Comment by zappolowski** [comment]
> unrelated question, should we add a completion for dunstify later?

Yes, this would make sense. I just added it for fish as that's what I'm using.

**Pull request merged**